### PR TITLE
feat(i18n): added Turkish translations

### PR DESCRIPTION
**Pull request by KutayGuler**
Added Turkish translations

### CODE REVIEW

**Comment by fabian-hiller** [comment]
Thank you for this PR! Do you know another native speaking developer that could review the translations for me?

**Comment by KutayGuler** [comment]
@behram Would you be up for the task?

File: packages/i18n/scripts/build.ts [hunk 1]
```diff
@@ -21,7 +22,7 @@ console.time('build');
 // Create languages array
 // Note: The language file `en` does not need to be added as the default
 // messages of Valibot are already in English
-const languages = [de, fr, ja, kr, nb, nl, pt, ro, ru, sl, uk, zhCN, zhTW];
+const languages = [de, fr, ja, kr, nb, nl, pt, ro, ru, sl, tur, uk, zhCN, zhTW];
```
Comment (behramcelen) [inline packages/i18n/scripts/build.ts:25]: @KutayGuler isn't is should be `tr` why tur?

**Review by behramcelen — APPROVED** [review]

**Comment by behramcelen** [comment]
@KutayGuler only I have above question beside of it all looks good, good job 👏

File: packages/i18n/scripts/build.ts (hunk 1 above)
Comment (behramcelen) [inline packages/i18n/scripts/build.ts:25]: @KutayGuler isn't is should be tr why tur?

Comment (KutayGuler) [reply packages/i18n/scripts/build.ts:25]: @behramcelen You are right, I got the wrong ISO code

**Pull request merged**